ParadiseSS13/Paradise
# fixes applying and removing pandora's hope effect

## What Does This PR Do
Fixes #20927 
Now Pandora's Hope effect can properly be applied to and removed from others.
Deleted some code that got doubled at some point and is unused.




## Why It's Good For The Game
effects being applied properly is probably something we want


## Testing
1. spawn human with shirt
2. apply pandora
3. they now have hope
4. remove pandora
5. they no longer have hope


## Changelog
:cl:
fix: Fixed the application and removal of the hope effect from pandora
/:cl:
